src/docs/product/data-management-settings/event-grouping/stack-trace-rules.mdx

```diff
@@ -204,50 +204,6 @@ For instance, the following marks all frames that are below a specific C++ names
 stack.function:myapplication::* +app
 ```
 
-#### Considerations for `sentry-cocoa`
```

Comment:
can we link to `/platform-redirect?next=/usage/in-app-frames` here

Reply:
This page currently only exist for Apple, so I added `/platforms/apple/usage/in-app-frames` instead. Do you think that's fine @untitaker?